### ISSUE TITLE
Missing no-duplicate-key implementation

### ISSUE THREAD

**Issue by michaeljota** [palantir/tslint#1599]
### Bug Report

- __TSLint version__: `4.0.0-dev0`
- __TypeScript version__: `2.0.3`
- __Running TSLint via__: CLI

#### Actual behavior
```shell
Error:
            Could not find implementations for the following rules specified in the configuration:
            no-duplicate-key
            Try upgrading TSLint and/or ensuring that you have all necessary custom rules installed.
            If TSLint was recently upgraded, you may have old rules configured which need to be cleaned up.
```
#### Expected behavior

To have a `no-duplicate-key` implementation

It's the rule `no-duplicate-key` have been remove or rename or something?

**Comment by adidahiya** [comment]
Yes, it was removed because this functionality is covered by the compiler now. Sorry I didn't get around to writing the changelog for v4.0.0-dev.0, but you can check out the milestone: https://github.com/palantir/tslint/milestone/26?closed=1

https://github.com/palantir/tslint/issues/885

**Comment by michaeljota** [comment]
Thanks! Will close this then. 😃

**Comment by dsifford** [comment]
@adidahiya How would one get around this issue if they were using the development branch + using `"extends": ["tslint:latest"]` in their `tslint.json`? Is that not possible?

**Comment by adidahiya** [comment]
@dsifford do you mean 4.0.0-dev.0? It was removed from the built-in configurations in that release too... https://github.com/palantir/tslint/blob/4.0.0-dev.0/src/configs/recommended.ts

**Comment by dsifford** [comment]
@adidahiya Hm.. Let me try again in a bit and report back.

Here's my build from earlier this afternoon that failed after bumping up to `@next`: https://travis-ci.org/dsifford/academic-bloggers-toolkit/builds/172345309

And here's the commit (you'll see 4.0.0-dev.0 listed in the `yarn.lock` file): https://github.com/dsifford/academic-bloggers-toolkit/pull/206/commits/10ab0bac9a0d2d362d7ac32ac789606c007ceec6

Let me know if I'm missing something obvious (which is generally the case!)

Also, please extend my thanks to the rest of the Palantir team for all their work on this. Such a critical part of my workflow! 💯 👍

**Comment by dsifford** [comment]
@adidahiya Sorry for the delay. 

This is an issue with `tslint-microsoft-contrib` and it appears to have been resolved in their master branch as of 11 days ago. Microsoft/tslint-microsoft-contrib#304

All good now 👍
